runtime/runtime/src/actions.rs

```diff
@@ -436,6 +436,7 @@ pub(crate) fn action_deploy_contract(
     apply_state: &ApplyState,
     current_protocol_version: ProtocolVersion,
 ) -> Result<(), StorageError> {
+    let _span = tracing::debug_span!(target: "vm", "action_deploy_contract").entered();
```

Comment:
```suggestion
    let _span = tracing::debug_span!(target: "runtime", "action_deploy_contract").entered();
```

this is still runtime